app/src/editor/shared/components/RenameInput.pcss

```diff
@@ -3,12 +3,15 @@
   color: inherit;
   border: none;
   background: none;
-  min-width: max-content; /* prevent name being squashed out */
+  min-width: available; /* prevent name being squashed out */
```

Comment:
note this isn't supported in all browsers and I think it's supposed to be `fill-available` https://developer.mozilla.org/en-US/docs/Web/CSS/min-width#Values

Reply:
Yep, that should be right